activity/settings/test.py

```diff
@@ -64,7 +67,7 @@
 
 # END DATABASE CONFIGURATION
 
-########## GOOGLE CLIENT CONFIG ###########
+# ######### GOOGLE CLIENT CONFIG ###########
```

Comment:
Let's stick to triple like `'''GOOGLE CLIENT CONFIG'''` quotation for such comments